README.md

```diff
@@ -151,29 +151,29 @@ and Charlie test accounts. In three separate terminals, initialize the data
 directories for the three Gossamer instances:
 
 ```sh
-./bin/gossamer init --force --config ./chain/westend-local/config-alice.toml
+./bin/gossamer init --force --config ./chain/westend_local/config-alice.toml
```

Comment:
why did we change to underscores for the folders? I like hyphens over underscores when it comes to folder names.